_pages/en_US/bannerbomb3-(legacy).txt

```diff
@@ -0,0 +1,50 @@
+---
+title: "BannerBomb3 (11.14 / 11.15)"
```

Comment:
title is not `BannerBomb3 (Legacy)`

well, it isn't all that important, but we did have an unwritten procedure to have the same title as the file name

Reply:
it's just to make it absolutely clear for the reader because of the risk involved but if we want to keep that convention we can